pkg/resource/alias/hooks.go

```diff
@@ -148,12 +163,43 @@ func (rm *resourceManager) setProvisionedConcurrencyConfig(
 	return nil
 }
 
-// getFunctionEventInvokeConfig will describe the fields that are
-// custom to the Alias resource
-func (rm *resourceManager) getFunctionEventInvokeConfig(
+func (rm *resourceManager) setFunctionEventInvokeConfigFromResponse(
+	ko *svcapitypes.Alias,
+	getFunctionEventInvokeConfigOutput *svcsdk.GetFunctionEventInvokeConfigOutput,
+	apiError error,
+) (err error) {
+
+	if apiError != nil {
+		if awserr, ok := ackerr.AWSError(apiError); ok && (awserr.Code() == "EventInvokeConfigNotFoundException" || awserr.Code() == "ResourceNotFoundException") {
+			ko.Spec.FunctionEventInvokeConfig = nil
+		} else {
+			return apiError
+		}
```

Comment:
The function is called `setFunctionEventInvokeConfigFromResponse` and is only supposed to set new fields, not handle errors they are not supposed to.

Reply:
Done.